eth-bytecode-db/eth-bytecode-db-server/docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: "3"
+services:
+  database:
+    image: postgres:15
+    container_name: 'eth-bytecode-db-postgres'
+    restart: always
+    environment:
+      POSTGRES_PASSWORD: ''
+      POSTGRES_USER: 'postgres'
+      POSTGRES_HOST_AUTH_METHOD: 'trust'
+      POSTGRES_DB: 'eth_bytecode_db'
+  #    ports:
+#      - "7432:5432"
+  smart-contract-verifier:
+    image: ghcr.io/blockscout/smart-contract-verifier:${SMART_CONTRACT_VERIFIER_DOCKER_TAG:-main}
+    container_name: 'eth-bytecode-db-verifier'
+    restart: always
+#    pull_policy: always
+    environment:
+      SMART_CONTRACT_VERIFIER__SERVER__HTTP__ENABLED: false
+      SMART_CONTRACT_VERIFIER__SERVER__GRPC__ENABLED: true
+      SMART_CONTRACT_VERIFIER__SERVER__GRPC__ADDR: 0.0.0.0:8044
+  eth-bytecode-db:
+    ## you can build an image locally, or use pre-built images from registry
+#    image: ghcr.io/blockscout/eth-bytecode-db:main
```

Comment:
you can leave image _and_ build fields
https://docs.docker.com/compose/compose-file/build/#illustrative-sample